.travis.yml

```diff
@@ -74,14 +71,10 @@ matrix:
     - os: osx
       language: generic
       env:
-        - PYTHON=3.6.6
```

Comment:
So the travis pip version uses pyenv to get the python version and it fails because "3.6" is too ambiguous. 

I noticed that the travis pip/openblas tended to be more delicate than the mkl version--especially for Mac. So I liked having separate ones. Not married to it, however.

Reply:
I think this can be dropped since python/pip is checked on azure for 3.5-3.7.

Reply:
I think this can be dropped since azure runs python/pip on OSX.  The only issue here is that OpenBlas is tied to the NumPy version, so different NumPy will get different OpenBlas.